docs/index.md

```diff
@@ -324,6 +324,59 @@ connector.Command(
 
 **Important note:** If the collection is empty, an `InvalidOperationException` will be thrown, since omitting the parameter entirely may not be valid (or intended) SQL.
 
+### Formatting SQL
+
+Typing parameter names in the SQL command text and parameters objects can sometimes be redundant. [`FormattableString`](https://docs.microsoft.com/en-us/dotnet/api/system.formattablestring) can be used
+to put the parameters in the SQL safely by using [`Sql.Format()`](Faithlife.Data.SqlFormatting/Sql/Format.md) to format a `FormattableString` into command text and parameters.
+
+```csharp
+var name = "First";
+var height = 6.875;
+connector.Command(Sql.Format(
+    $"insert into widgets (name, height) values ({name}, {height});"
+    )).Execute();
+```
+
+This is equivalent to:
+
+```csharp
+var name = "First";
+var height = 6.875;
+connector.Command(
+    "insert into widgets (name, height) values (@param0, @param1);",
```

Comment:
```suggestion
    "insert into widgets (name, height) values (@fdp0, @fdp1);",
```

Better to show the actual parameter names used?

Reply:
I wrote it that was at first, but the names are somewhat distracting, so I switched to `param`, which at least didn't leave me wondering what "fdp" means.